### DISCUSSION TITLE
Settings importer

### DISCUSSION THREAD

**Discussion by DennisWitte** [zed-industries/zed#14316] [Feature Requests]
I am really interested in trying ZED but everytime i do, i close it after a few seconds because all my handcrafted settings / keybinds from VS Code are no longer working. I have no idea how to switch between tabs or toggle a foldable reagion etc. When i try to change those settings i need to learn how ZED calls the specific function that i want to modify and everything gets annoying really fast. 

So my question is - are there any plans to have some kind of importer for VS code settings / keybinds to enable people to look into this editor with the best starting point possible?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
